### PR TITLE
Backend: Lesson Types 

### PR DESCRIPTION
**Pull request by shreyshrey1**
Best group

### CODE REVIEW

File: web_backend/src/schema/mutations/mutation.js [hunk 1]
```diff
@@ -17,7 +18,21 @@ const Mutation = new GraphQLObjectType({
           return s.save()
         } 
       },
-
+      createLesson: {
+        type: LessonType,
+        args: { id: { type: GraphQLID}, name: {type: GraphQLString}, quiz: { type: GraphQLString}, worksheets: { type: GraphQLString}, notes: { type: GraphQLString}},
```
Comment (tko22) [inline web_backend/src/schema/mutations/mutation.js]: For better readability, put dictionaries in a couple lines:
```Javascript
args: { 
   id: { type: GraphQLID}, 
   name: {type: GraphQLString},
   quiz: { type: GraphQLString}, 
   worksheets: { type: GraphQLString},
   notes: { type: GraphQLString}
}
```

File: web_backend/src/schema/queries/query.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
-import { GraphQLObjectType, GraphQLList } from 'graphql';
+import { GraphQLObjectType, GraphQLList, GraphQLID } from 'graphql';
```
Comment (tko22) [inline web_backend/src/schema/queries/query.js]: Good Job adding modules in the same line

File: web_backend/src/schema/types/LessonType.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import { GraphQLObjectType, GraphQLString, GraphQLID } from 'graphql';
+const mongoose = require('mongoose');
+
+const Lesson = new GraphQLObjectType({
+  name: 'Lesson',
+  description: 'Self Descriptive',
```
Comment (tko22) [inline web_backend/src/schema/types/LessonType.js]: Describe it

File: web_backend/src/schema/mutations/mutation.js [hunk 4]
```diff
@@ -17,7 +18,21 @@ const Mutation = new GraphQLObjectType({
...
+        resolve(root, { id, name, quiz, worksheets, notes}, ctx) {
+          const s = new Lesson({name, quiz, worksheets, notes})
+          return s.save()
+        }
+      },
+      deleteLesson: {
+        type: LessonType,
+        args: { id: {type: new GraphQLNonNull(GraphQLID)}},
```
Comment (tko22) [inline web_backend/src/schema/mutations/mutation.js]: same as above

File: web_backend/src/schema/mutations/mutation.js [hunk 5]
```diff
@@ -11,13 +12,38 @@ const Mutation = new GraphQLObjectType({
...
+          quiz: { type: GraphQLString}, 
+          worksheets: { type: GraphQLString}, 
+          notes: { type: GraphQLString}
+        },
+        resolve(root, { id, name, quiz, worksheets, notes}, ctx) {
+          const s = new Lesson({name, quiz, worksheets, notes})
+          return s.save()
+        }
+      },
+      // Doesn't work
```
Comment (CodeBrew28) [inline web_backend/src/schema/mutations/mutation.js]: Finish the delete functionality before merging

File: web_backend/src/schema/types/LessonType.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
+import { GraphQLObjectType, GraphQLString, GraphQLID } from 'graphql';
+const mongoose = require('mongoose');
+
+const Lesson = new GraphQLObjectType({
+  name: 'Lesson',
+  description: 'Schema design for lessons',
+  fields() {
+    return {
+      id: {
```
Comment (CodeBrew28) [inline web_backend/src/schema/types/LessonType.js]: format with name and type on the same line to maintain consistency.

**Pull request merged**